Resources/Prototypes/DeltaV/Entities/Clothing/Neck/cloaks.yml

```
@@ -30,4 +30,15 @@
   - type: Sprite
     sprite: DeltaV/Clothing/Neck/Cloaks/salvage.rsi 
   - type: Clothing
-    sprite: DeltaV/Clothing/Neck/Cloaks/salvage.rsi  
+    sprite: DeltaV/Clothing/Neck/Cloaks/salvage.rsi
```

```suggestion
    sprite: DeltaV/Clothing/Neck/Cloaks/salvage.rsi  
```
This should fix the random whitespace changes.